templates/rag-azure-search/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 LangChain, Inc.
```

Comment:
think this should be 2024 (@efriis think we need to update cli template)

Reply:
```suggestion
Copyright (c) 2024 LangChain, Inc.
```